### PR TITLE
Add: container-build workflow

### PR DESCRIPTION
**Pull request by pascalholthaus**
## What
Add: container-build-generic  workflow
Add: container-build-ghcr workflow

## Why
Makes the life easier to have it.

### CODE REVIEW

**Comment by y0urself** [comment]
Can we name it `docker-build-push` or `container-build-push`, so it gets clear, that it also pushes the image?

**Comment by bjoernricks** [comment]
I am in favor of container-build-push.

**Review by y0urself — APPROVED** [review]
Looks very good to me!

File: .github/workflows/container-build-push-docker.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        type: string
+        default: linux/amd64
+      build-context: 
+        description: "Path to image build context."
+        type: string
+        default: .
+      build-docker-file:
+        description: "Path to the docker file."
+        type: string
+        default: ./Dockerfile
```
Comment (bjoernricks) [inline .github/workflows/container-build-push-docker.yml:29]: It's missing the declaration of the requires secrets
```yml
secrets:
  DOCKERHUB_TOKEN:
    required: true
```

**Pull request merged**